### PR TITLE
v2.17.0

### PR DESCRIPTION
**Pull request by peachbits**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202490157836514

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,32 @@
 # edge-react-gui
 
+## 2.17.0 (2022-06-22)
+
+- Replace edge-currency-bitcoin with more performant edge-currency-plugins based on bitcoin.js
```
Comment (thehobbit85) [inline CHANGELOG.md:5]: Needs to mention that it also uses Blockbook instead of Electrum as the backend

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,32 @@
...
+- Refactor the SceneHeader into a memoized functional component
+- Refactor CurrencySettingsScene to pull directly from plugin currencyInfos
+- Update ExchangeQuoteComponent and remove unused DataRow and CardContent
+- Use theme hooks instead of higher order components
+- Fix deep link listener instability
+- Fix incorrect fiat display until restart for new wallets
+- Fix Invalid Password text alignment
+- Switch BSV block explorer to WhatsOnChain.com
+- Decrease swipeable row sensitivity
+- Add edge-currency-plugins v1.0.0
```
Comment (thehobbit85) [inline CHANGELOG.md:22]: This is identical to the first one, so combine them

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,32 @@
...
+- Remove edge-currency-bitcoin
+- Upgrade edge-core-js to v0.19.20
```
Comment (thehobbit85) [inline CHANGELOG.md:24]: You also need to include the changed from 0.19.19 since the last version of core on master in 0.19.18

**Review by thehobbit85 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (peachbits) [reply CHANGELOG.md:24]: It is included it's just one line

Comment (thehobbit85) [reply CHANGELOG.md:24]: My bad, I thought there were more changes. Never mind.

**Review by thehobbit85 — APPROVED** [review]

**Pull request merged**